anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-failover.adoc

```diff
@@ -21,6 +21,10 @@ To use the failover feature, do the following:
 * In Anypoint MQ:
 .. Create or modify a queue and xref:mq::mq-failover.adoc#enable-failover[enable failover on the queue].
 
+[NOTE]
+With failover enabled, the app can't consume directly from the fallback queue.
```

Comment:
app can't publish to fallback queue directly either.. so we should change this to publish or consume